src/components/DataSource/useActivatedUI.ts

```diff
@@ -20,3 +20,9 @@ export function useCurrentIdentity(noDefault?: boolean): Profile | null {
     const current = useValueRef(currentSelectedIdentity[getActivatedUI().networkIdentifier])
     return all.find(i => i.identifier.toText() === current) || (noDefault ? null : all[0])
 }
+export function useCurrentGroupsList() {
+    const groups = useGroupsList()
+    const currentIdentity = useCurrentIdentity()
+    if (!currentIdentity) return []
+    return groups.filter(x => x.identifier.ownerIdentifier.equals(currentIdentity.identifier))
```

Comment:
Note, invoke getter on `ownerIdentifier` may throw. You should check the `isVirtual` property before using it. (It maybe a type design failure maybe we should change the type `ownerIdentifier` to `null | ProfileIdentifier` instead of throw)